src/EditorFeatures/Test2/Rename/RenameEngineTests.VisualBasicConflicts.vb

```diff
@@ -3059,6 +3059,50 @@ End Class
                     result.AssertLabeledSpansAre("conflict", renameTo, RelatedLocationType.UnresolvedConflict)
                 End Using
             End Sub
+
+            <Fact>
+            <Trait(Traits.Feature, Traits.Features.Rename)>
+            <WorkItem(7440, "https://github.com/dotnet/roslyn/issues/7440")>
+            Public Sub RenameTypeParameterInPartialClass()
```

Comment:
What about type parameters in partial methods? Do we already have coverage of that scenario?